packages/expo-media-library/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### 💡 Others
 
+- Use `PHAssetCollectionSubtypeAny` subtype to avoid Recently Deleted album to show up ([#17561](https://github.com/expo/expo/pull/17561) by [@chuganzy](https://github.com/chuganzy))
```

Comment:
Could you move this entry to the "bug fixes" category? 🙏

Reply:
Sure thing! Pushing a new commit soon!